src/main/java/com/refinedmods/refinedstorage/RSBlocks.java

```diff
@@ -18,10 +17,9 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.function.Supplier;
 
 public final class RSBlocks {
-    private static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, RS.ID);
+    public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, RS.ID);
```

Comment:
Maybe we can move `DeferredRegister` to the `ColorMap` constructor so we can retain private?